### ISSUE TITLE
[3851622790 - Break In Story] Hello???!??? Have you heard of dropdowns????

### ISSUE THREAD

**Issue by RealPacket** [TrixAde/Proxima-Hub#2]
You already use dropdowns for some things, but when you're in the items tab, it's just all buttons,
TL;DR: Use dropdowns instead of using buttons for every item.
Even [my script](https://github.com/RealPacket/My-Scripts/blob/main/Games/BreakIn/KavoUI/BreakInStoryKavo.lua) which isn't obfuscated you clown 🤡 uses dropdown for selecting shit.
Just implement drop downs like you've done for some of the actions with multiple actions.
NOTE: I've not use the script yet, but I watched a video on it. I'll close this if you can add a dropdown for the items💀.
## Extra, not really related that much
Why are you obfuscating scripts on GitHub,
GitHub isn't supposed to be for scripts that are obfuscated, it's for NON obfuscated cool kid scripts.
If you really want to obfuscate a script, just put it on fucking pastebin or something else that isn't for open source💀.
After all, people who obfuscate just are clowns because they don't want people looking at their spaghetti code 💀

**Comment by RealPacket** [comment]
bro really obfuscated his loader script 💀
